variables.tf

```diff
@@ -4,29 +4,52 @@ variable "cluster_name" {
 }
 
 variable "base_domain" {
-  description = "The base domain used for Ingresses."
+  description = "The base domain used for Ingresses. If not provided, nip.io will be used taking the NLB IP address."
   type        = string
   default     = null
 }
 
-variable "kubernetes_version" {
-  description = "Kubernetes version to use for the SKS cluster."
+variable "cluster_zone" {
```

Comment:
The main point of this module is to deploy an SKS cluster, so I don't think the `cluster_` prefix in variables `cluster_zone` or `cluster_auto_upgrade` is very meaningful. We could same the same for `cluster_name`, but that's a variable we're already using on other modules.